vMenu/CommonFunctions.cs

```diff
@@ -1110,17 +1239,21 @@ public static void CycleThroughSeats()
         /// <param name="replacePrevious">Replace the previous vehicle of the player.</param>
         public static async void SpawnVehicle(string vehicleName = "custom", bool spawnInside = false, bool replacePrevious = false)
         {
+            string webhookVehicleName = vehicleName;
```

Comment:
Why not implement this in the actual SpawnVehicle function that spawns the actual vehicle?